src/main/resources/assets/computercraft/lua/rom/apis/keys.lua

```diff
@@ -53,6 +53,7 @@ for nKey, sKey in pairs( tKeys ) do
 	keys[sKey] = nKey
 end
 keys["return"] = keys.enter
+keys.scollLock = keys.scrollLock
```

Comment:
Probably should add a comment here explaining why the old name is kept, something like "typo kept to avoid a breaking change"

Reply:
I thought about it, I wasn’t sure if it was worth commenting though. I’ll add it when I get the chance.

Reply:
@KnightMiner I should mention I did this in the most recent commit.